RELEASES.md

```diff
@@ -0,0 +1,248 @@
+| Go API     | Unstable | 1.1 tentative     | [godoc](https://godoc.org/github.com/containerd/containerd) |
+| `ctr` tool | Unstable | Out of scope      | -             |
+
+From the version stated in the above table, that component must adhere to the
+stability constraints expected in release versions.
+
+Unless explicitly stated here, components that are called out as unstable or
+not covered may change in a future minor version. Breaking changes to
+"unstable" components will be avoided in patch versions.
+
+### GRPC API
+
+The primary product of containerd is the GRPC API. As of the 1.0.0 release, the
+GRPC API will not have any backwards incompatible changes without a _major_
+version jump.
+
+To ensure compatibility, we have collected the entire GRPC API symbol set into
+a single file. At each minor release of containerd, we will move the current
+`next.pb.txt` file to a file named for the minor version, such as `1.0.pb.txt`,
+enumerating the support services and messages. See [api/](api) for details.
+
+Note that new services may be added in _minor_ releases.
+
+#### Error Codes
+
+Error codes will not change in a patch release, unless a missing error code
+causes a blocking bug. Error codes of type "unknown" may change to more
+specific types in the future. Any error code that is not "unknown" that is
+currently returned by a service will not change without a _major_ release or a
+new version of the service.
+
+If you find that an error code that is required by your application is not
+well-documented in the protobuf service description or tested explicitly,
+please file and issue and we will clarify.
+
+#### Field Microformats
+
+Unless explicitly stated, the formats of certain fields may not be covered by
+this guarantee and should be treated opaquely. For example, don't rely on the
+format details of a URL field unless we explicitly say that the field will
+follow that format.
+
+### Go API
+
+The Go API, documented in
+[godoc](https://godoc.org/github.com/containerd/containerd), is currently
+considered unstable. It is currently recommended to vendor the necessary
+components to stabilize your project build.
+
+We intend to stabilize the API in a future release when more integrations have
+been carried out.
+
+Any changes to the API should be detectable at compile time, so upgrading will
+be a matter of fixing compilation errors and moving from there.
+
+### `ctr` tool
+
+The `ctr` tool provides the ability to introspect and understand the containerd
+API. At this time, it is not considered a primary offering of the project. It
+may be completely refactored or have breaking changes in _minor_ releases.
+
+We will try not break the tool in _patch_ releases.
+
+### Not Covered
+
+As a general rule, anything not mentioned in this document is not covered by
+the stability guidelines and may change in any release. Explicitly, this
+pertains to this non-inclusive list of components:
```

Comment:
I think you mean non-exhaustive? Or it might be easier to phrase it as: This explicitly pertains to, but is not limited to, the list of components below.